linera-base/src/data_types.rs

```diff
@@ -785,6 +785,15 @@ impl HashedBlob {
     pub fn id(&self) -> BlobId {
         self.id
     }
+
+    /// Loads a hashed blob from a string.
+    #[cfg(with_testing)]
+    pub fn test_blob(content: &str) -> Self {
```

Comment:
`from_string` ?

Reply:
I'm trying to follow the convention `CryptoHash` and `PublicKey` as suggested by @jvff: https://github.com/linera-io/linera-protocol/pull/2054#discussion_r1612139696